components/ExamQuizForm.tsx

```diff
@@ -58,14 +62,21 @@ const ExamQuizForm: React.FC<Props> = ({
         ))}
       </ul>
       <div className="flex justify-center flex-col sm:flex-row gap-4">
-        <Button type="submit" intent="primary" size="medium">
+        <Button 
+        type="button" 
+        intent="primary" 
+        size="medium"
+        onClick={() => {
+          reset(); handleSkipQuestion(currentQuestionIndex);
+        }}
```

Comment:
Please reformat your code manually, as you may have forgotten to run `npm run prepare`, causing it to not format while being committed.

Reply:
You're right, I skipped that part. Should be fixed in the latest commit